### PR TITLE
[WCAG 1.4.1] Update colors per new requirements

### LINKED ISSUE #34: Android guidelines - Update colors per new requirements
Define and add implementation examples (if applicable) for the given WCAG 2.2 Level A/AA points:

- [ ] [1.4.1](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=141)
- [ ] [1.4.3](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=143)
- [ ] [2.3.1](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=231)

Keep in mind that some points are already defined in this repository but it might be that they are still for WCAG 2.1 or they are only written for Android traditional view system. We also need to add compose support where that makes sense.

Your main reference points should be:
1. The [WCAG website](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa) 
2. The iOS implementation of the same point in this repository

**If you find some useful articles, videos, or any resource please share it with the rest of us in our dedicated slack channel.
For any questions you can ask Renato, Petra and Majcen**

### PR DESCRIPTION
**Pull request by zavrskina**
**Description:**

Added changes close #34 :
- 1.4.1 - Added _Use of colors section_ even though the guideline is dependant on accessible design since the same is mentioned in the official Android docs
- 1.4.3 - Minimum contrast guideline should be satisfied on the design level and there are no specifics for Android there so I've added it to the "others" section
- 2.3.1 - the section already existed so I just unified the title with the latest changes

Feel free to share if you think any of the sections should be removed or have any other ideas or suggestions.

### CODE REVIEW

File: docs/guidelines/platforms/android/guideline_percievable_android.md [hunk 1]
```diff
@@ -291,6 +291,32 @@ The content provided in your app must be easily understandable to all users. Tha
...
+
+:no_entry_sign: **Failures**
+
+- An important difference between elements is stressed only with colors.
+
+_Important to note is that this guideline primarily depends on accessible design._
+
+## Other perceivable guidelines
+
+This section includes guidelines that may not apply to the Android platform or fall under the mobile team’s responsibilities. However, please keep in mind that these guidelines still need to be met."
```
Comment (sttihan) [inline docs/guidelines/platforms/android/guideline_percievable_android.md:316]: Extra `"`?
```suggestion
This section includes guidelines that may not apply to the Android platform or fall under the mobile team’s responsibilities. However, please keep in mind that these guidelines still need to be met.
```

**Review by sttihan — APPROVED** [review]
Awesome! 🚀 I think it's okay to include the "Use of Color" guideline, even though it depends entirely on the design, but I think it doesn't hurt to be aware of it.

File: docs/guidelines/platforms/android/guideline_operable_android.md [hunk 2]
```diff
@@ -116,23 +116,23 @@ Moving, blinking, or scrolling, or auto-updating content in the app.
 
 ---
 
-## Seizures and Physical Reactions
+## Seizures and Physical Reactions (WCAG 2.3.1 - Level A)
```
Comment (Aksi0m) [inline docs/guidelines/platforms/android/guideline_operable_android.md:119]:
```suggestion
## Seizures and Physical Reactions (WCAG 2.3)
```

File: docs/guidelines/platforms/android/guideline_operable_android.md [hunk 3]
```diff
@@ -116,23 +116,23 @@ Moving, blinking, or scrolling, or auto-updating content in the app.
...
 
 Do not design content in a way that is known to cause seizures or physical reactions.
 
 ### Three Flashed or Below Threshold
```
Comment (Aksi0m) [inline docs/guidelines/platforms/android/guideline_operable_android.md:123]:
```suggestion
### Three Flashed or Below Threshold (WCAG 2.3.1 - Level A)
```

**Review by Aksi0m — APPROVED** [review]
Looks good! I like that we have this use of color defined in more detail. It will stress the importance of this a bit more. Also, we do not want to rely on designers only (same goes the other way around).

**Pull request merged**